bfunwind/src/dwarf4.cpp

```diff
@@ -30,14 +30,22 @@
 template<typename T> T static get(char **p)
 { auto v = *(T *)(*p); (*p) += sizeof(T); return v;}
 
-#define if_opcode(a,b) \
+#define if_cfa(a,b) \
     if (opcode == a) \
     { \
         log("  %s: ", #a); \
         b \
         return; \
     }
 
+#define if_opcode(a,b) \
+    if (opcode == a) \
+    { \
+        log("    - %s: ", #a); \
+        b \
+        continue; \
```

Comment:
if_opcode is in a while loop. The goal of the macro is to simplify the amount of cut / past there is. Looking at Apple's implementation, there are so many CFA and Expression opcodes, writing "continue" / "return" every time distracts from the point of the logic, but yeah, normally I would agree. For an example, please see: https://github.com/llvm-mirror/libunwind/blob/master/src/DwarfParser.hpp#L435